trust/Makefile

```diff
@@ -50,13 +49,21 @@ _helm_init:
 install-cert-manager-trust: _helm_init
 	@echo 'Installing Jetstack cert-manager trust operator.....'
 	@cp templates/values.yaml artifacts/trust/values.yaml 
-
-	@helm upgrade \
+ 	
+	
+#	@helm upgrade \
 	    --install -f artifacts/trust/values.yaml cert-manager-trust jetstack/cert-manager-trust \
   		--namespace jetstack-secure \
   		--version ${JS_CERT_MANAGER_TRUST_VERSION} \
 		--wait
 
+	@helm upgrade \
+		--install -f templates/values.yaml trust-manager \
+		--namespace jetstack-secure \
+		oci://eu.gcr.io/jetstack-secure-enterprise/charts/trust-manager \
+	    --version ${JS_CERT_MANAGER_TRUST_VERSION} \
+		--wait
+
 _remove-policy-and-rbac:
 	@kubectl delete -f config/cert-policy-rbac.yaml || true
```

Comment:
can you delete this line considering you have merged crp and rbac ?

Reply:
Resolved in pull request 17